okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -510,7 +511,7 @@ private void relayToRequest(HttpClientResponse res, ProxyContext pc,
   }
 
   private void log(HttpClientRequest creq) {
-    logger.debug("{} {}", creq.getMethod().name(), creq.getURI());
+    logger.debug("{} {}", () -> creq.getMethod().name(), () -> creq.getURI());
```

Comment:
This is not needed if using
```
if (! logger.isDebugEnabled()) {
  return;
}
```
at the begin of this method.

Reply:
OK.. Lower coverage though ..